bool winStatus
uint8 playerId

# Add more data type that comes out of GameController
